src/rules/readme-lists-correct-yarn-version.ts

```diff
@@ -0,0 +1,32 @@
+import { buildRule } from './helpers';
+import { RuleName } from './types';
+
+export default buildRule({
+  name: RuleName.ReadmeListsCorrectYarnVersion,
+  description:
+    'Does the README conform by recommending the correct Yarn version to install?',
+  dependencies: [RuleName.RequireReadme, RuleName.AllYarnModernFilesConform],
+  execute: async ({ template, project, pass, fail }) => {
+    const entryPath = 'README.md';
+
+    const fileContentInTemplate = await template.fs.readFile(entryPath);
+    const fileContentInProject = await project.fs.readFile(entryPath);
+
+    const match = fileContentInTemplate.match(
+      /Install \[Yarn [^[\]]+\]\([^()]+\)/u,
```

Comment:
Nit: Might be simpler to write it this way, using a non-greedy modifier:

```suggestion
      /Install \[Yarn .+?\]\(.+?\)/u,
```

At least I think that should do the same thing

Reply:
You're right! I get a bit paranoid when it comes to regexes, I guess, but this should work just fine. Applied.